lib/omnibus/config.rb

```diff
@@ -443,6 +443,16 @@ def reset!
     # @return [String, nil]
     default(:solaris_linker_mapfile, "files/mapfiles/solaris")
 
+    # Architecture to target when building on windows.  This option
+    # should affect the bit-ness of Ruby and DevKit used, the platform of
+    # any MSIs generated and package dlls being downloaded.
+    #
+    # See the windows_x86? and windows_x64? software definition dsl
+    # methods.
+    #
+    # @return [:x86, :x64]
+    default(:windows_arch, :x86)
```

Comment:
Ah fair enough.  And I guess because omnibus-chef has a Gemfile.lock, Manhattan won't pick up this change until we update it which we can do so in the same pull-request during which we add that option to the config file.